cmd/influx/cli/cli.go

```diff
@@ -798,6 +798,8 @@ func (c *CommandLine) ExecuteQuery(query string) error {
 			err = ctx.Err()
 			if err == context.Canceled {
 				err = errors.New("aborted by user")
+			} else {
+				err = errors.New("unknown error")
```

Comment:
Should this instead by "no data received"? It seems like it describes what the usual cause is (server crashed and returned no data) while not necessarily declaring that it was the exact cause because we don't really know if that's what happened.

Reply:
I thought of printing something like "No data: Probably server crashed", because this is the usual case. "No data received" seems to be too ambigous to me, though.